Qiqqa/Documents/PDF/PDFControls/PDFReadingControl.xaml

```diff
@@ -33,13 +33,6 @@
                     <local_gui:AugmentedToggleButton x:Name="ButtonInk"/>
                     <local_gui:AugmentedToggleButton x:Name="ButtonTextSentenceSelect"/>
                     <local_gui:AugmentedToggleButton x:Name="ButtonCamera"/>
-                    <local_gui:AugmentedToolBarButton x:Name="ButtonExplore"/>
-                    <local_gui:AugmentedToolBarPopup x:Name="ButtonExplorePopup">
-                        <StackPanel>
-                            <local_gui:AugmentedToolBarButton x:Name="ButtonExpedition" CaptionDock="Right" />
-                            <local_gui:AugmentedToolBarButton x:Name="ButtonExploreInBrainstorm" CaptionDock="Right" />
-                        </StackPanel>
-                    </local_gui:AugmentedToolBarPopup>
```

Comment:
Just curious: why did you move it?

Reply:
By its meaning citing is closer to selecting text and taking an image snapshot than to exploring with expedition. Also citing is more important than exploring (therefore closer to the beginning).

Reply:
Wouldn't always agree with the "more important" bit there (depends on your work/task; if you're not writing papers, citing is not a much needed feature), but otherwise: yes.

Like I said: just wondering.

Picked up some of your easy changes already, will merge the rest later this week.